include/geos/operation/polygonize/PolygonizeGraph.h

```diff
@@ -201,28 +201,30 @@ class GEOS_DLL PolygonizeGraph: public planargraph::PlanarGraph {
 	 * from the graph, so that there is always a next dirEdge.
 	 *
 	 * @param startDE the DirectedEdge to start traversing at
-	 * @param edgesInRing : the DirectedEdges that form a ring will
-	 *                      be pushed here.
+	 * @result a vector of the DirectedEdge that form a ring
 	 */
-	static void findDirEdgesInRing(PolygonizeDirectedEdge *startDE,
-		std::vector<planargraph::DirectedEdge*>& edgesInRing);
+	std::vector<planargraph::DirectedEdge*>
+	findDirEdgesInRing(PolygonizeDirectedEdge *startDE) const;
```

Comment:
This forces a new allocation of a vector, and a copy of it, whereas the output parameter avoids that

Reply:
I can make the move explicit if you want